docs/keycloak/manual-client-secret-setup-pl.md

```
@@ -0,0 +1,92 @@
+Jeśli korzystasz z IntelliJ IDEA, możesz to zrobić wygodnie, klikając jeden przycisk z panelu „Services” lub z poziomu edytora – jak pokazano na poniższym screenie:
+
+📸 *Przycisk uruchamiający Keycloak w IntelliJ:*
+![Keycloak startup screenshot](./images/run-keycloak-instance.png)
+
+Aby sprawdzić, czy inicjalizacja Keycloak przebiegła pomyślnie, możesz otworzyć przeglądarkę i przejść pod adres:
+
+[http://localhost:8081](http://localhost:8081)
+
+Jeśli wszystko działa poprawnie, powinieneś zobaczyć ekran logowania do konsoli administracyjnej Keycloak.
+
+📸 *Keycloak login form:*<br>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: br

(MD033, no-inline-html)